test/unit/lib/adapters/config/playwright.ts

```diff
@@ -0,0 +1,208 @@
+import path from 'node:path';
+import sinon from 'sinon';
+
+import {PlaywrightConfigAdapter, DEFAULT_BROWSER_ID, type PwtConfig, type PwtProject} from '../../../../../lib/adapters/config/playwright';
```

Comment:
Those `../../..` are ugly as hell =( Would be nice to import them via `@site/lib/adapters`. Absolutely not suggesting to implement it in this PR, just thinking out loud, maybe we need an issue for this...

Reply:
> Those ../../.. are ugly as hell =(

Yes, I suffer from this when I write tests. We can write path like - `'lib/adapters/config/playwright'`due to the use of `app-module-path`, but he loses types on the way so I don't use it.

In general, it seems that we just need to move to jest and put the tests near to the code, then there will be no problems.

> Would be nice to import them via @site/lib/adapters

I didn't understand how it would work? What the `@site` means?